lib/buildsystems/autotools.rb

```diff
@@ -34,9 +32,6 @@ def self.install
   end
 
   def self.check
-    if run_tests
-      puts 'Testing with make check.'.orange
```

Comment:
Needed to confirm tests are running?

Reply:
We could compromise here and only display with the `-v` or `--verbose` option.

Reply:
My (petty) issue was with the extra line in the buildsystem, not with the display while building. Your logic makes sense, I'll re-add this line (and the others like it)